src/components/Graphs/ClusterGraph.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import * as d3 from 'd3';
+import './ClusterGraph.css';
+
+interface Props {
+    data: any;
+    width: number;
+    height: number;
+}
+
+export default function ClusterGraph({ width, height, data }: Props) {
+    const maxSize = Math.max.apply(
+        Math,
+        data.map(function (o) {
+            return o.size;
+        })
+    );
```

Comment:
Use array destructuring for this.
```suggestion
    const maxSize = Math.max(...data.map((o) => o.size));
```